pages/functional/js/factorial.js

```diff
@@ -1,67 +1,31 @@
 import { parseNumber } from "../../js/helper.js";
 
 /**
- * Count factorial number from the given "n" value using loop way.
- *
- * @param {number} n
+ * Factorial Using Loop
+ * @param {*} n 
+ * @returns 
  */
-function countFactorialUsingLoop(n) {
-  n = parseNumber(n);
+function factorialUsingLoop (n) {
+  let formula = n; 
 
-  let result = 1;
-
-  for (let index = n; index > 0; index--) {
-    result *= index;
+  for (let i = 1; i < n; i += 1) {
+      formula *= (n-i);
   }
-
-  return result;
+  
+  return formula;
 }
 
-/**
- * Count factorial number from the given "n" value using recursive way.
- *
- * @param {number} n
- * @returns {number}
- */
-function countFactorialUsingRecursive(n) {
-  n = parseNumber(n);
-
-  if (n < 2) {
-    return 1;
-  }
-
-  return n * countFactorialUsingRecursive(n - 1);
-}
+console.log (factorialUsingLoop (6));
 
 /**
- * Count factorial number from the given "n" value.
- *
- * @param {number} n
- * @param {"loop" | "recursive"} method
- * @throws {Error}
- */
-function countFactorial(n, method) {
-  if (method == "loop") {
-    return countFactorialUsingLoop(n);
-  } else if (method == "recursive") {
-    return countFactorialUsingRecursive(n);
-  } else {
-    throw new Error("Method must be loop or recursive.");
+*Factorial Using Recursive 
+* @param {number} n 
+* @returns 
+*/
+function factorialUsingrecursive (n) {
+  return n <= 1 
+  ? 1
+  : factorialUsingrecursive (n - 1) * n; 
   }
-}
-
-document.getElementById("form").addEventListener("submit", function (event) {
-  event.preventDefault();
 
```

Comment:
Where is the JavaScript DOM object calling? Don't remove that as it will be used to handle the HTML input.